### PR TITLE
Add a page about pkg-config

### PR DESCRIPTION
**Pull request by lazka**
This focuses on things that are special/different compared to Linux.

### CODE REVIEW

File: web/docs/pkgconfig.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+```bash
+prefix=/ucrt64
+includedir=${prefix}/include
+libdir=${prefix}/lib
+```
+
+As you can see `/ucrt64` is not a proper Windows path, but that's not a problem
+because by default `prefix` will be ignored, or rather re-defined by
+pkgconf/pkg-config based on the location of the .pc file itself. It will strip
+of `/<...>/pkgconf` from the directory of the `.py` file and use the resulting
```
Comment (jeremyd2019) [inline web/docs/pkgconfig.md:28]:
```suggestion
of `/<...>/pkgconfig` from the directory of the `.pc` file and use the resulting
```

?

Comment (lazka) [reply web/docs/pkgconfig.md]: thanks

Comment (jeremyd2019) [reply web/docs/pkgconfig.md]: oops, just noticed that line probably should have started with "off" rather than "of" too

Comment (lazka) [reply web/docs/pkgconfig.md]: thanks, fixed

**Pull request merged**